projects/plugins/protect/src/js/data/scan/use-scan-status-query.ts

```diff
@@ -10,6 +10,37 @@ import {
 import { ScanStatus } from '../../types/scans';
 import { QUERY_SCAN_STATUS_KEY } from './../../constants';
 
+export const isRequestedScanNotStarted = ( status: ScanStatus ) => {
+	// If the scan status is not "idle", always return the fresh API data
+	if ( status.status !== 'idle' ) {
+		return false;
+	}
+
+	// Retrieve last scan timestamp from localStorage and convert to number
+	const lastRequestedScanTimestamp = Number( localStorage.getItem( 'last_requested_scan' ) );
+
+	// If there is no stored timestamp, return the API data
+	if ( ! lastRequestedScanTimestamp ) {
+		return false;
+	}
+
+	// Check if the last scan request is more than 5 minutes old
+	if ( lastRequestedScanTimestamp < Date.now() - 5 * 60 * 1000 ) {
+		return false;
+	}
+
+	// Convert the lastChecked date string to a Unix timestamp
+	const lastCheckedTimestamp = new Date( status.lastChecked + ' UTC' ).getTime();
+
+	// Check if the scan request is completed based on the last checked time
+	const isScanCompleted = lastCheckedTimestamp > lastRequestedScanTimestamp;
```

Comment:
```suggestion
	if ( status.status !== 'idle' ) {
		return false;
	}

	const lastRequestedScanTimestamp = Number( localStorage.getItem( 'last_requested_scan' ) );

	if ( ! lastRequestedScanTimestamp ) {
		return false;
	}

	if ( lastRequestedScanTimestamp < Date.now() - 5 * 60 * 1000 ) {
		return false;
	}

	const lastCheckedTimestamp = new Date( status.lastChecked + ' UTC' ).getTime();

	const isScanCompleted = lastCheckedTimestamp > lastRequestedScanTimestamp;
```
Nit - these comments don't add any new information, they just describe the next line of code (i.e. "If the scan status is not idle" vs "if ( scan.status !== 'idle' ) {").

Additionally, since we have abstracted this into a method called `isRequestedScanNotStarted`, we should remove comments referencing the returning of API data now that that we're returning a boolean value describing something independent from API data handling specifically.